build.sbt

```diff
@@ -5,13 +5,11 @@ version := "1.0"
 
 scalaVersion in ThisBuild := "2.11.11"
 
-resolvers ++= Seq("Guardian Bintray" at "https://dl.bintray.com/guardian/editorial-tools")
-
 scalacOptions ++= Seq(
   "-deprecation",
   "-Ywarn-unused-import")
 
-lazy val awsVersion = "1.11.77"
+lazy val awsVersion = "1.11.678"
```

Comment:
Strangely, .77 caused problems on deployment – we got ```
2021-04-30 10:36:48,194 [main] WARN  com.amazonaws.util.EC2MetadataUtils:387 - Unable to retrieve the requested metadata.
2021-04-30 10:36:48,199 [main] WARN  com.amazonaws.util.EC2MetadataUtils:387 - Unable to retrieve the requested metadata.
2021-04-30 10:36:48,207 [main] ERROR configuration-magic:10 - Impossible to create the amazon ec2 client
java.lang.NullPointerException: null
```
Downgrading to the lowest version that supports IMDSv2 seems to have solved the problem.